### PR TITLE
Abitily to configure inside paths where templates could be located

### PR DESCRIPTION
**Pull request by sars**
Could you approve this please?

### CODE REVIEW

File: lib/angular-rails-templates/engine.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ class Engine < ::Rails::Engine
     config.angular_templates = ActiveSupport::OrderedOptions.new
     config.angular_templates.module_name    = 'templates'
     config.angular_templates.ignore_prefix  = ['templates/']
+    config.angular_templates.inside_paths  = [find_root('.').join('app', 'assets')]
```
Comment (pitr) [inline lib/angular-rails-templates/engine.rb]: Did you mean `Rails.root.join('app', 'assets')`?

**Comment by pitr** [comment]
This looks good! Thanks @sars. Can you add info about `inside_paths` to README

**Comment by msyesyan** [comment]
@pitr I need this, too. Would you please merge it to master? because I want to load assets/templates files from a third-part gem, the original method will only work for Rails.root/assets/templates, thanks very much

**Comment by morizotter** [comment]
:+1:

**Comment by pitr** [comment]
v0.2.0 is released. Thanks @sars

**Comment by sars** [comment]
thanks, @pitr 
sorry for not answering for a long time

**Comment by pitr** [comment]
no problem @sars, thanks for contributing this PR

**Pull request closed without merging**